webrender/src/device.rs

```diff
@@ -701,11 +702,19 @@ impl Device {
         _file_changed_handler: Box<FileWatcherHandler>,
         cached_programs: Option<Rc<ProgramCache>>,
     ) -> Device {
-        let max_texture_size = gl.get_integer_v(gl::MAX_TEXTURE_SIZE) as u32;
+        let mut max_texture_size = 0;
```

Comment:
wouldn't it be easier to just go with `let mut max_texture_size = [0]`, followed by `&mut max_texture_size` and then taking `[0]` from it as opposed to relying on external `ref_slice` crate?

Reply:
Maybe, which way do you prefer? I don't particularly care either way, I went for the simpler code that made the less invasive changes, not having to do `[0]` everywhere.